### PR TITLE
Onboarding - Store Profiler: Hide the option "CBD and other hemp-derived products"

### LINKED ISSUE #573: Onboarding - Store Profiler: Hide the option "CBD and other hemp-derived products"
WordPress.com doesn't support CBD, so the option `CBD and other hemp-derived products` should be removed from the Industry step in the eCommerce plan.

### PR DESCRIPTION
**Pull request by octaedro**
Fixes #573

As WordPress.com doesn't support CBD, this PR removes the option `CBD and other hemp-derived products` from the Industry step in the eCommerce plan.

### Screenshots
screenshot-three wordpress test-2020 09 10-10_49_02

### Detailed test instructions:
To test this PR it's necessary to use a site with [wc-calypso-bridge](https://github.com/Automattic/wc-calypso-bridge) installed.
- Go to the industry step in the store profiler (URL: `/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard&step=industry`)
- Verify the option `CBD and other hemp-derived products` is not visible anymore.

- Be sure the tests are running ok.

_The changes related to the effects this modification will have on the payments methods will be handled in the WooCommerce Admin [PR 5124](https://github.com/woocommerce/woocommerce-admin/pull/5124)_

### Changelog Note:
Fix: Removed CBD from onboarding for eCommerce plan

### CODE REVIEW

File: tests/unit-tests/test-class-wc-calypso-bridge-filters.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ * Tests for WC_Calypso_Bridge_Filters
+ */
+
+class WC_Calypso_Bridge_Filters_Test extends WC_Calypso_Bridge_Test {
+	/**
+	 * Test removing `CBD and other hemp-derived products` option from industries list
+	 *
+	 */
+	public function test_remove_not_allowed_industries() {
```
Comment (becdetat) [inline tests/unit-tests/test-class-wc-calypso-bridge-filters.php:11]: This should be two tests as you're testing two different things

File: wc-calypso-bridge.php [hunk 2]
```diff
@@ -52,6 +52,10 @@ function wc_calypso_bridge_is_ecommerce_plan() {
 	}
 }
 
+// Filters we want to add for ecommerce plan.
+require_once dirname( __FILE__ ) . '/includes/class-wc-calypso-bridge-filters.php';
```
Comment (becdetat) [inline wc-calypso-bridge.php:56]: Isn't this already done in `bootstrap.php`?

**Comment by becdetat** [comment]
Everything tests well, I just had a couple of comments in the code.

**Comment by octaedro** [comment]
Hi @becdetat,
I answered the comments you made. Could you give this PR another look?

**Review by timmyc — APPROVED** [review]
Thanks so much for adding some tests Fernando, this is testing well for me in the OBW also.

**Pull request merged**